### PR TITLE
Update html_helper.php

### PR DESCRIPTION
**Pull request by avegacms**
Added hreflang parameter for link_tag function for calling <link href="https://domain.com/en" hreflang="x-default" rel="alternate"> and etc. And added checks for tags with rel expressions for  'alternate','canonical'  in this case attribute type dont use

Each pull request should address a single issue and have a meaningful title.

**Description**
Explain what you have changed, and why.

**Checklist:**
- [ ] Securely signed commits
- [ ] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

---------Remove from here down in your description----------

**Notes**
- Pull requests must be in English
- If the PR solves an issue, reference it with a suitable verb and the issue number
(e.g. fixes <hash>12345)
- Unsolicited pull requests will be considered, but there is no guarantee of acceptance
- Pull requests should be from a feature branch in the contributor's fork of the repository
  to the develop branch of the project repository

### CODE REVIEW

**Comment by avegacms** [comment]
Added hreflang parameter for link_tag function for calling and etc. And added checks for tags with rel expressions for 'alternate','canonical' in this case attribute type dont use

File: system/Helpers/html_helper.php [hunk 1]
```diff
@@ -306,7 +308,17 @@ function link_tag($href = '', string $rel = 'stylesheet', string $type = 'text/c
 			$link .= 'href="' . $href . '" ';
 		}
 
-		$link .= 'rel="' . $rel . '" type="' . $type . '" ';
+		if( $hreflang !== '' )
```
Comment (MGatner) [inline system/Helpers/html_helper.php:311]: `if ($hreflang !== '')`

File: system/Helpers/html_helper.php [hunk 2]
```diff
@@ -306,7 +308,17 @@ function link_tag($href = '', string $rel = 'stylesheet', string $type = 'text/c
...
+		{
+		    $link .= 'hreflang="' . $hreflang .'" ';
+		}
+
+		$link .= 'rel="' . $rel . '" ';
+
+		if( !in_array($rel, ['alternate','canonical']) )
```
Comment (MGatner) [inline system/Helpers/html_helper.php:318]: `if (! in_array($rel, ['alternate', 'canonical']))`

File: system/Helpers/html_helper.php (hunk 1 above)
Comment (avegacms) [reply system/Helpers/html_helper.php]: Ok, i fixed this

File: system/Helpers/html_helper.php (hunk 2 above)
Comment (avegacms) [reply system/Helpers/html_helper.php]: Ok, i fixed this

**Comment by kenjis** [comment]
@avegacms I sent a PR #7022 to fix `link_tag()`.
If it breaks your app, please let us know.

**Pull request merged**